src/ui/public/agg_types/metrics/lib/parent_pipeline_agg_controller.js

```diff
@@ -0,0 +1,63 @@
+const aggFilter = ['!top_hits', '!percentiles', '!percentile_ranks', '!median', '!std_dev'];
+
+const parentPipelineAggController = function ($scope) {
+
+  $scope.safeMakeLabel = function (agg) {
+    try {
+      return agg.makeLabel();
+    } catch (e) {
+      return '- agg not valid -';
+    }
+  };
+
+  $scope.$watch('responseValueAggs', updateOrderAgg);
+  $scope.$watch('agg.params.metricAgg', updateOrderAgg);
+
+  $scope.$on('$destroy', function () {
+    if ($scope.aggForm && $scope.aggForm.agg) {
+      $scope.aggForm.agg.$setValidity('bucket', true);
+    }
+  });
+
+  // Returns true if the agg is not compatible with the terms bucket
+  $scope.rejectAgg = function (agg) {
+    // aggFilter elements all starts with a '!'
+    // so the index of agg.type.name in a filter is 1 if it is included
+    return Boolean(aggFilter.find((filter) => filter.indexOf(agg.type.name) === 1));
+  };
+
+  function checkBuckets() {
+    const buckets = $scope.vis.aggs.filter(agg => agg.schema.group === 'buckets');
+    const bucketHasType = buckets.length && buckets[0].type;
+    const bucketIsHistogram = bucketHasType && ['date_histogram', 'histogram'].includes(buckets[0].type.name);
+    const canUseAggregation = buckets.length === 1 && bucketIsHistogram;
+    if ($scope.aggForm.agg) $scope.aggForm.agg.$setValidity('bucket', canUseAggregation);
+    if (canUseAggregation) {
+      if (buckets[0].type.name === 'histogram') {
+        buckets[0].params.min_doc_count = 1;
```

Comment:
Can you explain `min_doc_count` to me?  Does it mean essentially `showExtendedBounds`?  I see the extended bounds thing in histogram and this value is used to conditionally show and output them.  Looks like this number is only ever 0 or 1 too?  One thing I don't understand is why the value is flipped here in histogram.js:

```
        write: function (aggConfig, output) {
          if (aggConfig.params.min_doc_count) {
            output.params.min_doc_count = 0;
          } else {
            output.params.min_doc_count = 1;
          }
        }
```

I don't see where `shouldShow` in histogram.js would ever return false because I only see number and date fields in the drop down.  Well, except when no field is selected, but thats an error anyway, so why ever hide it?Anyway, this is somewhat outside the scope of this PR so if you aren't sure, it can be something to explore later. Just trying to understand it.